src/utils/FormattingSettingsUtils.ts

```diff
@@ -38,7 +38,7 @@ export function getPropertyValue(slice: SimpleSlice, value: any, defaultValue: a
         return { value: (value as Fill)?.solid.color };
     }
 
-    if ((slice as ItemDropdown)?.items) {
+    if ((slice as ItemDropdown)?.items && slice.type !== visuals.FormattingComponent.FlagsSelection) {
```

Comment:
instead of checking it's not flagsSelection, can we check that the type is specifically equal to ItemDropdown before the check if it has items